File: Assets/My Assets/Scripts/UI/Narrative/Flashing Text.js
var ColorA : Color;
var ColorB : Color;
var blendTime : float = 0.5;
var myText : UI.Text;
function Start(){
	if(myText == null){
		myText = GetComponent(UI.Text);
	}
}
function Update(){
	myText.color = Color.Lerp(ColorA, ColorB, Mathf.PingPong(Time.time*blendTime, 1));
}

File: Assets/My Assets/Scripts/UI/Narrative/Flashing Text.cs
using UnityEngine;
using System.Collections;

[System.Serializable]
public partial class Flashing Text : MonoBehaviour
{
    public Color ColorA;
    public Color ColorB;
    public float blendTime;
    public UI.Text myText;
    public virtual void Start()
    {
        if (this.myText == null)
        {
            this.myText = GetComponent(UI.Text);
        }
    }

    public virtual void Update()
    {
        this.myText.color = Color.Lerp(this.ColorA, this.ColorB, Mathf.PingPong(Time.time * this.blendTime, 1));
    }

    public Flashing Text()
    {
        this.blendTime = 0.5f;
    }

}